azure-mgmt-resources/src/test/java/com/microsoft/azure/management/resources/core/TestUtilities.java

```diff
@@ -27,4 +39,32 @@ public static void sleep(int milliseconds, boolean isRecordMode) {
     public static String createPassword() {
         return "12NewPA$$w0rd!";
     }
+
+    private static String sshPublicKey;
+
+    public static String createSshPublicKey() {
```

Comment:
@xccc-msft I've added a method to generate public ssh key.

The public key will appear in response json. Hope CredScan is ok with it.